embedded-compositor/embedded-compositor.pro

```diff
@@ -22,6 +22,7 @@ HEADERS +=  embeddedshellextension.h \
             sortfilterproxymodel.h
 
 OTHER_FILES += \
+    main.qml \
```

Comment:
What kind of main.qml file is it, under qml/ folder are also a main.qml

Reply:
It looks like that this is the first of the POC at the beginning